Packs/ILert/Integrations/ILert/README.md

```diff
@@ -0,0 +1,96 @@
+Alert and notify users using iLert
```

Comment:
iLert is a modern uptime platform with advanced on-call and alerting features.

Alert and notify users using iLert.
This integration was integrated and tested with API version 1 of iLert.

## Configure iLert on Cortex XSOAR